backend/app/adapter/sqldb/changelog.go

```diff
@@ -89,6 +89,34 @@ WHERE "%s" = $1;
 	return err
 }
 
+// UpdateChange updates an existing change in change_log table
```

Comment:
```suggestion
// UpdateChange updates an existing change in change_log table.
```
Forgot to add this, based on this https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences